storm-core/test/clj/org/apache/storm/scheduler_test.clj

```diff
@@ -262,20 +262,15 @@
 
 (deftest test-sort-slots
   ;; test supervisor2 has more free slots
-  (is (= '(["supervisor2" 6700] ["supervisor1" 6700]
-           ["supervisor2" 6701] ["supervisor1" 6701]
-           ["supervisor2" 6702])
-         (sort-slots [["supervisor1" 6700] ["supervisor1" 6701]
-                      ["supervisor2" 6700] ["supervisor2" 6701] ["supervisor2" 6702]
-                      ])))
+  (is (= "[supervisor2:6700, supervisor1:6700, supervisor2:6701, supervisor1:6701, supervisor2:6702]"
+         (.toString (EvenScheduler/sortSlots [(WorkerSlot. "supervisor1" 6700) (WorkerSlot. "supervisor1" 6701)
+                      (WorkerSlot. "supervisor2" 6700) (WorkerSlot. "supervisor2" 6701) (WorkerSlot. "supervisor2" 6702)
+                      ]))))
   ;; test supervisor3 has more free slots
-  (is (= '(["supervisor3" 6700] ["supervisor2" 6700] ["supervisor1" 6700]
-           ["supervisor3" 6703] ["supervisor2" 6701] ["supervisor1" 6701]
-           ["supervisor3" 6702] ["supervisor2" 6702]
-           ["supervisor3" 6701])
-         (sort-slots [["supervisor1" 6700] ["supervisor1" 6701]
-                      ["supervisor2" 6700] ["supervisor2" 6701] ["supervisor2" 6702]
-                      ["supervisor3" 6700] ["supervisor3" 6703] ["supervisor3" 6702] ["supervisor3" 6701]
-                      ])))
+  (is (= "[supervisor3:6700, supervisor2:6700, supervisor1:6700, supervisor3:6701, supervisor2:6701, supervisor1:6701, supervisor3:6702, supervisor2:6702, supervisor3:6703]"
```

Comment:
Same here toString is not ideal for testing.  It can change way too frequently and is meant to be user consumable not machine consumable.